src/librustc_hir/lang_items.rs

```diff
@@ -109,6 +187,12 @@ impl<CTX> HashStable<CTX> for LangItem {
     }
 }
 
+impl<CTX> HashStable<CTX> for LangItemRecord {
+    fn hash_stable(&self, _: &mut CTX, hasher: &mut StableHasher) {
+        ::std::hash::Hash::hash(self, hasher);
+    }
```

Comment:
Is this implementation correct? `LangItemRecord` can contain a `DefId` for which `hash` is not stable.

Reply:
Good catch.